### PR TITLE
Add max_length to Email (254) & update .gitignore

### PR DESCRIPTION
**Pull request by lud4ik**
According to links
http://stackoverflow.com/questions/386294/what-is-the-maximum-length-of-a-valid-email-address
http://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690
limit is 254.
In current version I can do:

```
t.Email().check('x' * 100000 + '@ukr.net')
```

and it's valid.

### CODE REVIEW

**Comment by asvetlov** [comment]
LGTM!

File: trafaret/__init__.py [hunk 1]
```diff
@@ -627,10 +629,12 @@ class Email(String):
         r')@(?P<domain>(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)$)'  # domain
         r'|\[(25[0-5]|2[0-4]\d|[0-1]?\d?\d)(\.(25[0-5]|2[0-4]\d|[0-1]?\d?\d)){3}\]$', re.IGNORECASE)  # literal form, ipv4 address (SMTP 4.1.3)
     min_length = None
-    max_length = None
+    max_length = MAX_EMAIL_LEN
```
Comment (Deepwalker) [inline trafaret/__init__.py:632]: I don't see why we need these code lines at all. String class will set up max and min length anyway, so this is just redundant.

**Comment by lud4ik** [comment]
Thanks!

**Comment by Deepwalker** [comment]
Thank you for pull. But next time please add test ;)

**Pull request merged**